### DISCUSSION TITLE
What's the difference between react and next?

### DISCUSSION THREAD

**Discussion by rickywilliam1010** [community/community#37882] [Other Feature Feedback, Questions, & Ideas] [answered]
I have a problem about to migrate.

**Comment by ghost** [comment, marked as answer]
Next JS is a framework that holds React on its base, while React JS is an open-source JavaScript library, developed and maintained by Facebook. Next JS is used to create web applications and performs server-side rendering, whereas React JS focuses on rendering towards the DOM.

**Comment by snowMan128** [comment]
React, developed by Facebook, is an open-source library that allows you to build UI components.
ReactJS performs client-side rendering.
